doc/nrf/known_issues.rst

```diff
@@ -1022,6 +1066,23 @@ DRGN-15382: The SoftDevice Controller cannot be qualified on nRF52832
 
   **Workaround:** Upgrade to v1.5.1 or use the master branch.
 
+.. rst-class:: v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-13029: The application will not immediately restart a connectable advertiser after a high duty cycle advertiser times out.
```

Comment:
This bug is already mentioned below as "Wait time required after a directed high duty cycle advertiser times out"
Can you update that entry instead?